troi/patches/top_recording_similar_recordings.py

```diff
@@ -0,0 +1,133 @@
+from datetime import datetime 
+from collections import defaultdict
+from urllib.parse import quote
+import requests
+
+import click
+
+from troi import Element, Artist, Recording, Playlist, PipelineError
+import troi.listenbrainz.stats
+import troi.filters
+import troi.sorts
+import troi.musicbrainz.recording_lookup
+import troi.musicbrainz.mbid_mapping
+
+
+@click.group()
+def cli():
+    pass
+
+class LookupSimilarRecordingsElement(Element):
+    """ Lookup similar recordings.
+
+        Given a list of recordings, this element will output recordings that are similar to the ones passed in.
+    """
+
+    def __init__(self):
+        super().__init__()
+
+    @staticmethod
+    def inputs():
+        return [Recording]
+
+    @staticmethod
+    def outputs():
+        return [Recording]
+
+    def read(self, inputs):
+
+        sim = []
+        for recording in inputs[0]:
+            url = f"https://labs.api.listenbrainz.org/similar-recordings/json"
+            r = requests.get(url, { "recording_mbid": recording.mbid, "algorithm": "session_based_days_7_session_300" })
```

Comment:
can use count = 2 to avoid the server from doing extra work since we are only using the first 2 recs.